curve25519-dalek/src/backend/serial/risc0/field.rs

```diff
@@ -0,0 +1,212 @@
+//! Field arithmetic modulo \\(p = 2\^{255} - 19\\), using \\(32\\)-bit
+//! limbs with \\(64\\)-bit products.
+
+use core::fmt::Debug;
+use core::ops::Neg;
+use core::ops::{Add, AddAssign};
+use core::ops::{Mul, MulAssign};
+use core::ops::{Sub, SubAssign};
+
+use crypto_bigint::{risc0, Encoding, U256};
+use subtle::{Choice, ConditionallySelectable};
+
+#[cfg(feature = "zeroize")]
+use zeroize::Zeroize;
+
+/// A `FieldElementR0` represents an element of the field
+/// \\( \mathbb Z / (2\^{255} - 19)\\). `FieldElementR0`
+/// leverages RISC0's big integer accelerated zkvm circuit.
```

Comment:
nit: "RISC Zero" is the preferred way to refer to us in natural language
```suggestion
/// leverages RISC Zero's big integer accelerated zkvm circuit.
```